neptun-cli/src/main.rs

```diff
@@ -136,18 +137,18 @@ fn main() {
             })) => {
                 let mut b = [0u8; 1];
                 if sock2.recv(&mut b).is_ok() && b[0] == 1 {
-                    println!("BoringTun started successfully");
+                    println!("NepTUN started successfully");
                     exit(first_child_exit_code)
                 } else {
-                    eprintln!("BoringTun failed to start");
+                    eprintln!("NepTUN failed to start");
```

Comment:
nit: Are we going for `NepTun` similar to `BoringTun` or `NepTUN` ?

Reply:
NepTUN